seed/challenges/03-front-end-libraries/jquery.json

```diff
@@ -145,7 +145,7 @@
       "tests": [
         "assert($(\"button\").hasClass(\"animated\") && $(\"button\").hasClass(\"bounce\"), 'message: Use the jQuery <code>addClass&#40&#41</code> function to give the classes <code>animated</code> and <code>bounce</code> to your <code>button</code> elements.');",
         "assert(!code.match(/class.*animated/g), 'message: Only use jQuery to add these colors to the element.');",
-        "assert(/'/g.test(code) === false , 'message: Only use double quotes in your jQuery code, as is stated in the jQuery style guide.');"
+        "assert(/'/g.test(code) === false , 'message: Try using double quotes in your jQuery code, as described in challenge description');"
```

Comment:
I vote we change the note on this challenge to `Though not required, it is recommended to use double quotes on selectors in jQuery as a best practice of the library.` Then, we remove this test. Otherwise, as @techstonia points out, we would need to enforce it on all challenges and since it's not actually required by the library, that seems to be unnecessary.